cpu/stm32/periph/dma.c

```diff
@@ -196,6 +192,7 @@ static IRQn_Type dma_get_irqn(int stream)
     if (stream < 7) {
         return ((IRQn_Type)((int)DMA1_Channel1_IRQn + stream));
     }
+#if !defined(CPU_FAM_STM32F3)
 #if defined(CPU_FAM_STM32F1)
```

Comment:
Seems redundant. If `CPU_FAM_STM32F1` is defined, `CPU_FAM_STM32F3` won't be defined and vice-versa.

Reply:
There is a #else to #if defined(CPU_FAM_STM32F1) which result in `error: 'DMA2_Channel1_IRQn' undeclared` at compilation time. Same thing 9 lines later. This drop the code up to line 212.

Reply:
Ah ok - better add comments to the `#endif` so it's apparent what the matching `#if` was.

Reply:
Comment added.